packages/web/hooks/use-estimate-tx-fees.ts

```diff
@@ -26,34 +26,70 @@ async function estimateTxFeesQueryFn({
   messages,
   apiUtils,
   accountStore,
+  sendToken,
 }: {
   wallet: AccountStoreWallet<[OsmosisAccount, CosmosAccount, CosmwasmAccount]>;
   accountStore: AccountStore<[OsmosisAccount, CosmosAccount, CosmwasmAccount]>;
   messages: EncodeObject[] | undefined;
   apiUtils: ReturnType<typeof api.useUtils>;
+  sendToken?: {
+    balance: CoinPretty;
+    amount: CoinPretty;
+  };
 }): Promise<QueryResult> {
   if (!messages) throw new Error("No messages");
```

Comment:
If we're going to throw an error immediately at runtime if `messages` is undefined, why don't we just change the `messages` type to not be capable of being undefined at build time?